src/sedtrails/particle_tracer/time.py

```diff
@@ -47,23 +47,30 @@ def __post_init__(self):
         if not isinstance(self.start_time, int):
                 raise TypeError(f"Expected 'start_time' to be an int, got {type(self.start_time).__name__}")
 
-    def get_current_time(self) -> np.datetime64:
+    def get_current_time(self, delta_seconds: int = 0) -> np.datetime64:
         """
-        Returns the current time as a numpy.datetime64 object.
+        Returns the current time in simulation as a numpy.datetime64 object,
+        optionally including an additional delta in seconds.
+
+        Parameters
+        ----------
+        delta_seconds : int, optional
+            Additional seconds to add to the current simulation time (default is 0).
 
         Returns
         -------
         numpy.datetime64
             The current time in the simulation.
         """
-        return self._reference_date_np + self.time_as_timedelta64()
+        total_seconds = self.start_time + delta_seconds
+        return self._reference_date_np + np.timedelta64(total_seconds, 's')
 
-    def get_seconds_since_reference(self) -> int:
+    def get_seconds_since_reference(self) -> str:
         """
-        Returns the number of seconds since the reference date/time.
+        Returns the number of seconds since the reference date/time in a 
+        human readable format
         """
-        # Will complete once I can clarify the different between reference_date and simulation start
-        pass   
+        return str(self.get_current_time())   
```

Comment:
I think I was misunderstood. I did not mean to say to modify this method. Rather, we need to add a new method for the human readable conversion, for example: `get_datetime(self)`, this should return the date and time (Gregorian time) in the same format as in our CF convention of choice. We should leave the get_seconds_since_reference() as it was, and return the number of seconds.

Reply:
I will have read the full code on the file, so I need a few minutes. But, with the changes we discussed today, we aren't redefining the methods (what they should do), only making changing the implementation. I will get back with concrete examples.